arangod/RestHandler/RestDocumentHandler.cpp

```diff
@@ -45,7 +45,12 @@ using namespace arangodb::rest;
 
 RestDocumentHandler::RestDocumentHandler(application_features::ApplicationServer& server,
                                          GeneralRequest* request, GeneralResponse* response)
-    : RestVocbaseBaseHandler(server, request, response) {}
+    : RestVocbaseBaseHandler(server, request, response) {
+  if(request->requestType() == rest::RequestType::POST
```

Comment:
Isn't this the wrong way round? Short requests are allowed to be done directly, longer not? I would probably take 1024 bytes as the limit, rather than 120.